cantools/db/formats/dbc.py

```diff
@@ -264,6 +264,45 @@ def _create_grammar():
                         - scolon)
     signal_type.setName(SIGNAL_TYPE)
 
+    message_add_sender = Group(Keyword('BO_TX_BU_')
```

Comment:
Use the already declared variable `MESSAGE_TX_NODE`. Apply similar change to other already declared keywords.